wrapanapi/systems/ec2.py

```diff
@@ -886,13 +885,15 @@ def delete_sqs_queue(self, queue_url):
     def get_all_unused_loadbalancers(self):
         return [
             loadbalancer for loadbalancer
-            in self.elb_connection.get_all_load_balancers()
-            if not loadbalancer.instances]
+            in self.elb_connection.describe_load_balancers().get("LoadBalancerDescriptions")
```

Comment:
Looks like the API response is changing here, from a list of objects to a list of dictionaries.  The expected format of each element in the list returned should be put in a docblock.